### PR TITLE
Changes menu for side nav

### PR DESCRIPTION
**Pull request by paullewis**

### CODE REVIEW

File: src/client/styles/_components/_side-nav.scss [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/**
+ *
+ * Copyright 2016 Google Inc. All rights reserved.
```
Comment (piecioshka) [inline src/client/styles/_components/_side-nav.scss]: Maybe `2017`?

File: src/server/apps/dynamic.js [hunk 2]
```diff
@@ -89,7 +90,14 @@ dynamic.get('/', (req, res) => {
     inlines
   });
 
-  res.status(200).render('home', viewOptions);
+  res.render('home', viewOptions, (err, html) => {
+    if (err) {
+      res.status(500).send('Fail');
```
Comment (piecioshka) [inline src/server/apps/dynamic.js]: @paullewis why you don't exit function with `return` statement?

File: src/server/apps/dynamic.js [hunk 3]
```diff
@@ -124,7 +136,14 @@ dynamic.get('/*', (req, res) => {
     return res.status(200).render('listing', viewOptions);
   }
 
-  res.status(200).render('video', viewOptions);
+  res.render('video', viewOptions, (err, html) => {
+    if (err) {
+      res.status(500).send('Fail');
```
Comment (piecioshka) [inline src/server/apps/dynamic.js]: The same here. Where is `return` statement?

Comment (paullewis) [inline src/server/apps/dynamic.js]: Good call

File: src/server/apps/dynamic.js (hunk 2 above)
Comment (paullewis) [inline src/server/apps/dynamic.js]: Good call

File: src/client/styles/_components/_side-nav.scss (hunk 1 above)
Comment (paullewis) [inline src/client/styles/_components/_side-nav.scss]: Mmhmm! I used my code from last year, but sure.

**Comment by paullewis** [comment]
Should be fixed in 9f073ac

File: src/server/apps/dynamic.js (hunk 3 above)
Comment (WORMSS) [inline src/server/apps/dynamic.js]: Many Eyes, Makes Solid Code..

**Pull request merged**